docs/source/en/perf_infer_gpu_one.mdx

```diff
@@ -36,20 +38,45 @@ Below are some notes to help you use this module, or follow the demos on [Google
 
 ### Requirements
 
-- Make sure you run that on NVIDIA GPUs that support 8-bit tensor cores (Turing, Ampere or newer architectures - e.g. T4, RTX20s RTX30s, A40-A100).
+- Make sure you run on NVIDIA GPUs that support 8-bit tensor cores (Turing, Ampere or newer architectures - e.g. T4, RTX20s RTX30s, A40-A100).
```

Comment:
(unrelated to this PR but from the version `0.37.0` of `bitsandbytes` all GPU models should be supported, check: https://github.com/TimDettmers/bitsandbytes/releases/tag/0.37.0